db/migrate/20221212225921_enable_sidebar_and_chat.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class EnableSidebarAndChat < ActiveRecord::Migration[7.0]
+  def up
+    result = execute <<~SQL
+      SELECT created_at
+      FROM schema_migration_details
+      ORDER BY created_at
+      LIMIT 1
+    SQL
+
+    # keep sidebar legacy and chat disabled for for existing sites
+    if result.first['created_at'].to_datetime < 1.hour.ago
```

Comment:
Sorry can I trouble you to write a spec for this migration since this is some what high risk and I want to make sure we've covered all the possible combinations. You can refer to an existing example of a migration spec [here](https://github.com/discourse/discourse/pull/19336/commits/741be98280f37789d19a04efb9aeab02c30be872).